circle.yml

```diff
@@ -27,9 +28,11 @@ jobs:
     # TODO: Investigate whether we can do some caching to speed up
     # this step, e.g. of /tmp/go-android/pkg.
     - run:
-        working_directory: ../client/shared
         environment:
+          LOCAL_CLIENT: 0
           LOCAL_KBFS: 1
```

Comment:
Ah, you mean in https://github.com/keybase/kbfs-server/pull/559/files . That's because the Docker doesn't have a kbfs checkout, whereas here in CircleCI there is one.